data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
Hey @smcgregor, I tried a [part](https://github.com/SummerWish/privly-chrome/blob/embed-posting/javascripts/content_scripts/embeded_posting.js#L193) of SummerWish's proposal(the link you gave me). It didn't work but I think that's mostly because I missed out on some other key details of his larger proposal. In order to completely try/test it out, I need to implement nearly every part of SummerWish's "Embed Posting" proposal. So this is what I plan --
* Work on the tests(both Integration and Unit tests) for PR #4 and #5 without changing the current posting process.
* Once I am done with the tests, I'll begin with changing the posting process as done by SummerWish, which will hopefully not need the large library.